training-service/requirements.

```diff
@@ -0,0 +1,48 @@
+apispec==5.1.1
```

Comment:
wrong requirements.txt export

1. we only have a requirements.txt in the root folder, please delete the requirements. in the test folder
2. requirements. != requirements.txt (wrong file format export?), command for the export should be ```pip freeze > requirements.txt``` from the training-service directory

Reply:
okay got it